resources/keymap_de_win.map

```diff
@@ -0,0 +1,147 @@
+CPC_0	SDLK_0
```

Comment:
Because I integrated this file in the other PR, I think you'll have to do a
`git pull upstream master`
or something like that.

You may need to do:
`git remote add upstream git://github.com/ColinPitrat/caprice32.git
git fetch upstream`
before.